content/blog/2024/oci-manifest-capability.md

```diff
@@ -92,17 +98,26 @@ fn validate(payload: &[u8]) -> CallResult {
             warn!(LOG_DRAIN, "cannot unmarshal resource: this policy does not know how to evaluate this resource; accept it");
         }
     }
+    match  manifest("ghcr.io/kubewarden/policies/psp-user-group:v0.1.2") {
+        Ok(manifest) => {
+            print_manifest(manifest)
+        }
+        Err(error) => {
+            warn!(LOG_DRAIN, "cannot fetch container image manifest: {:?}", error);
 
+        }
+    }
     kubewarden::accept_request()
 }
-
 ```
 
 This straightforward code iterates through the containers utilized by a Pod. It
 checks each container's manifest, and if the manifest conforms to an OCI (Open
 Container Initiative) image manifest, it logs this information. In cases where
 the manifest does not match an OCI image manifest, the code instead logs the
-platforms detailed in the OCI index manifest.
+platforms detailed in the OCI index manifest. After that, the policy request
+the manifest for other OCI artifact. In this example, we are asking for the OCI
+manifest for the `psp-user-group` Kubewarden policy.
```

Comment:
This should be removed if we don't have the example of asking for the  `psp-user-group` manifest.

I think it's ok just having the nginx example.

Reply:
We are requesting the manifest of the policy. See this code in the end of the validation function:

```rust
    match  manifest("ghcr.io/kubewarden/policies/psp-user-group:v0.1.2") {
        Ok(manifest) => {
            print_manifest(manifest)
        }
        Err(error) => {
            warn!(LOG_DRAIN, "cannot fetch container image manifest: {:?}", error);

        }
    }
```

Reply:
Latest example remove this code anyways. Marking as resolved